### PR TITLE
Fix #923 added scripts required for debuging and setting up

### LINKED ISSUE #923: Create a dev script to bring up the entire 0chain
Whenever we do a clean deploy it takes a lot of time to edit files fire commands using the documentation.

The scope of this script is currently limited to the dev env and that is why we do not consider the scenarios such as

initial requirements are not installed
network configuration required in macOS and windows are not required to be set

This script should be downloaded and executed as simple one script to bring up the entire infra

### PR DESCRIPTION
**Pull request by samtholiya**
Tasks to complete before merging PR:
- [x]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [x]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [x]  Merge your system tests PR to master BEFORE merging this PR  
  
Associated PRs (Link as appropriate):
...

### CODE REVIEW

File: 0chain_dev_deployment.sh [hunk 1]
```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
```
Comment (kushthedude) [inline 0chain_dev_deployment.sh:1]: Will this script run on single terminal? How would I ensure that all 3 miners or 2 sharders are up? How can I see their console logs & outputs if I setup chain using this script?

File: 0chain_dev_deployment.sh [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    case "${flag}" in
+        s) sharderCount=${OPTARG};;
+        m) minerCount=${OPTARG};;
+        b) blobberCount=${OPTARG};;
+    esac
+done
+
+git clone https://github.com/0chain/0chain.git
+git clone https://github.com/0chain/blobber.git
+git clone https://github.com/0chain/0dns.git
```
Comment (kushthedude) [inline 0chain_dev_deployment.sh:18]: Can we have a check here if these directories are present already?

File: 0chain_dev_deployment.sh (hunk 1 above)
Comment (samtholiya) [reply 0chain_dev_deployment.sh:1]: `docker logs -f <container_name>` to check logs 
`docker ps` to check if all containers are up (we can update the script to do that as well good idea I guess 🤔  )
This does not clutter up the console as well as allows us to check the logs we care about

File: 0chain_dev_deployment.sh (hunk 2 above)
Comment (samtholiya) [reply 0chain_dev_deployment.sh:18]: if it is already present these commands anyway do no harm
image

**Review by kushthedude — COMMENTED** [review]
Looks good to me, can we move this into a top-level `scripts` folder?

Saves time for me to run similar commands 3-4 times

**Review by cnlangzi — APPROVED** [review]
LGTM

**Review by kushthedude — APPROVED** [review]

**Pull request merged**